README.md

```diff
@@ -127,9 +127,10 @@ between client-go versions.
 | client-go 2.0  | Kubernetes main repo, 1.5 branch     | = -                           |
 | client-go 3.0  | Kubernetes main repo, 1.6 branch     | = -                           |
 | client-go 4.0  | Kubernetes main repo, 1.7 branch     | = -                           |
-| client-go 5.0  | Kubernetes main repo, 1.8 branch     | ✓                             |
+| client-go 5.0  | Kubernetes main repo, 1.8 branch     | = -                           |
```

Comment:
https://github.com/kubernetes/client-go/releases/tag/v5.0.0 was released on Oct 21. It's been >6 months since then. The deprecation policy says that we will maintain for "at least 6 months".

Also see https://github.com/kubernetes/client-go/pull/391#discussion_r178746452